### DISCUSSION TITLE
Wireshark fails to detect packets properly

### DISCUSSION THREAD

**Discussion by T-449** [open-quantum-safe/oqs-demos#292] [Q&A]
Hi,

I was trying to monitor the packet transfer between client and server for PQ implementation of OpenVPN. But for some reason there are cases when Wireshark is unable to detect the correct message type specifically when there is fragmentation or when TLS1.2 is used. This happens irrespective of the underlying transport layer protocol.
![Screenshot from 2024-08-14 11-53-08](https://github.com/user-attachments/assets/d1e98966-578b-4b0b-95ae-338ad290dce9)
![Screenshot from 2024-08-14 11-53-28](https://github.com/user-attachments/assets/aee71588-cba8-4fd2-aaee-e0abcd6d82a9)

If u take a look at the screenshots the message type should have been client hello for those messages. I can't even expand into the TLS layer because it fails to parse the data.

**Comment by planetf1** [comment]
@t-449 does enabling [IP Reassembly](https://wiki.wireshark.org/IP_Reassembly) in wireshark help ? 

Obviously there are other potential discussion points about the particular network scenario, options to reduce chance of defragmentation etc.

**Comment by alraddady** [comment]
Hi @T-449, Wireshark has been updated and fixed—could you test it again and let us know if it's resolved? However, I apologize as it won’t work on macOS due to GUI compatibility issues, which will be fixed soon.
